docx/oxml/text/paragraph.py

```diff
@@ -83,13 +84,13 @@ def style(self, style):
     def r_lst_recursive(self):
         """
         Override xmlchemy generated list of runs to include runs from
-        hyperlinks.
+        hyperlinks and content controls.
         """
 
         def get_runs(elem):
             for child in elem:
                 if child.tag == qn('w:r'):
                     yield child
-                elif child.tag == qn('w:hyperlink'):
+                elif child.tag in (qn('w:hyperlink'), qn('w:sdt'), qn('w:sdtContent')):
```

Comment:
PEP-8. Lines should be of len < 80.

Reply:
I think we extended line len to 99 chars for better readability